tests/e2e_tests.py

```diff
@@ -39,20 +46,28 @@ def test_explicit_paths(link_localtime_mock, get_timezone_for_ip_mock):
             FAKE_TIMEZONE, zoneinfo_path, localtime_path,
         ),
     )
+    assert_true(
+        export_etc_timezone_mock.called_once_with(
+            FAKE_TIMEZONE, etc_timezone_path,
+        ),
+    )
 
 
 @mock.patch('tzupdate.get_timezone_for_ip')
+@mock.patch('tzupdate.export_etc_timezone')
 @mock.patch('tzupdate.link_localtime')
-def test_explicit_ip(_, get_timezone_for_ip_mock):
+def test_explicit_ip(_, export_etc_timezone_mock, get_timezone_for_ip_mock):
     ip_addr = '1.2.3.4'
     get_timezone_for_ip_mock.return_value = FAKE_TIMEZONE
     args = ['-a', ip_addr]
     tzupdate.main(args)
     assert_true(get_timezone_for_ip_mock.called_once_with(ip_addr))
+    assert_true(export_etc_timezone_mock.called)
```

Comment:
The assert_true has no bearing on that happening or not, only the mock does.

That is, you can add the mock, you don't have to then assert `.called` on it.